Gemfile

```diff
@@ -50,13 +50,16 @@ gem "httpclient"
 gem "view_component", require: "view_component/engine"
 
 # Database based asynchronous priority queue system
+gem "daemons"
```

Comment:
What are we using this for?

Reply:
To [run delayed jobs](https://github.com/collectiveidea/delayed_job#running-jobs)